docs/contents/cloud-spanner.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "Spanner",
```

Comment:
This file controls what appears on the sidebar table of contents in the [docs site](https://googlecloudplatform.github.io/google-cloud-php/#/docs/google-cloud/v0.27.0/servicebuilder). We use the short service names (i.e. "Logging" and "Spanner") in the table of contents, similar to other languages.